.gitignore

```diff
@@ -48,4 +48,21 @@ node_modules/
 .next
 
 # MACOS sillyness
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# WebStorm project files
```

Comment:
I think we should be able to just ignore all .idea directories

Reply:
Edited; replaced the lines with .idea/ (this should be the syntax according to documentation, feel free to correct...)

Reply:
I think it should be ok now